packages/component/src/ReactDocumentPictureInPicture.tsx

```diff
@@ -131,8 +131,8 @@ const ReactDocumentPictureInPicture = forwardRef<
 
 
     let featureUnavailableReason: FeatureUnavailableReasonEnum | null = (() => {
-        const isUsingSecureProtocol = window.location.protocol === 'https:';
-        if (isUsingSecureProtocol === false) return FeatureUnavailableReasonEnum.USING_UNSECURE_PROTOCOL;
+        const isSecureContext = window.isSecureContext;
+        if (isSecureContext === false) return FeatureUnavailableReasonEnum.USING_UNSECURE_PROTOCOL;
```

Comment:
I've avoided renaming `USING_UNSECURE_PROTOCOL` to avoid the breaking change for now, but ideally this would be renamed.